Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -348,7 +353,8 @@ class BrowserViewController: UIViewController {
                     if let readerMode = self.tabManager.selectedTab?.getHelper(name: ReaderMode.name()) as? ReaderMode where readerMode.state == .Active {
```

Comment:
@thebnich 
This is here because this seemed to be the best place to run something when we have switched back to the web view after showing the home panels. Is it causing problems? Maybe we should introduce a callback for it?